From 8f338639366f5f549d9cb491fb2eee574733b188 Mon Sep 17 00:00:00 2001
From: Jorge Ruesga <jorge@ruesga.com>
Date: Mon, 25 May 2015 23:14:36 +0200
Subject: [PATCH 07/20] email: linkify phone numbers

Allow linkify and open phone numbers in emails
JIRA: CML-117

Change-Id: I039d2d412a7d8dfd0dcc5a78dd214a282811ba6b
Signed-off-by: Jorge Ruesga <jorge@ruesga.com>
---
 src/com/android/mail/providers/Message.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/com/android/mail/providers/Message.java b/src/com/android/mail/providers/Message.java
index 331be1b31..c51fc20d9 100644
--- a/src/com/android/mail/providers/Message.java
+++ b/src/com/android/mail/providers/Message.java
@@ -716,7 +716,8 @@ public class Message implements Parcelable, HtmlMessage {
             body = bodyHtml;
         } else if (!TextUtils.isEmpty(bodyText)) {
             final SpannableString spannable = new SpannableString(bodyText);
-            Linkify.addLinks(spannable, Linkify.EMAIL_ADDRESSES | Linkify.WEB_URLS);
+            Linkify.addLinks(spannable, Linkify.EMAIL_ADDRESSES | Linkify.WEB_URLS
+                    | Linkify.PHONE_NUMBERS);
             body = Html.toHtml(spannable);
         }
         return body;
-- 
2.17.1

